hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdapterApiUtils.java

```diff
@@ -44,6 +51,117 @@
  */
 public class ProtocolAdapterApiUtils {
 
+    private static final Logger LOG = LoggerFactory.getLogger(ProtocolAdapterApiUtils.class);
+    private static final String DEFAULT_SCHEMA = "{\n" +
```

Comment:
suggestion(non-blocking): This is generally not easy to manipulate. Are there any other methods, e.g. to read from the disc?